lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -119,6 +119,28 @@
     - Maximum number of CloudFormation events to fetch from a stack when creating or updating it.
     default: 200
     version_added: "2.7"
+  backoff_delay:
+    description:
+    - Number of seconds to wait for the next retry.
+    default: 3
+    version_added: "2.8"
+    type: int
+    required: False
+  backoff_retries:
+    description:
+    - Number of times to retry operation.
+    - AWS API throttling mechanism fails Cloudformation module so we have to retry a couple of times.
+    default: 10
+    version_added: "2.8"
+    type: int
+    required: False
+  backoff_max_delay:
```

Comment:
pedantic, but here and in the module params below, keep the order consistent. Thankfully alphasort will group the 'delay' by each other: backoff_delay, backoff_max_delay, backoff_retries.